docs/modules/ROOT/partials/proc-manage-environment-variables.adoc

```diff
@@ -10,21 +11,51 @@ OpenShift web console::
 ifdef::apicurio-registry[]
 . Select the *Installed Operators* tab, and then *{registry} Operator*.
 endif::[]
-ifdef::rh-service-registry[]
+ifdef::service-registry[]
 . Select the *Installed Operators* tab, and then *Red Hat Integration - Service Registry Operator*.
 endif::[]
 . On the *Apicurio Registry* tab, click the `ApicurioRegistry` CR for your {registry} deployment.
-. On the main overview page, view the `managedResources` section, which contains the name of the `Deployment` managed by the Operator to deploy your {registry} instance.
-. Find that `Deployment` in the *Workloads* > *Deployments* in the left menu.
-. Select the `Deployment` with the correct name, and select the *Environment* tab.
-. You can add or modify your environment variable to the *Single values (env)* section.
-. Click *Save* at the bottom.
+. Click on the *YAML* tab and then edit the `spec.configuration.env` section as needed, for example:
++
+[source,yaml]
+----
+apiVersion: registry.apicur.io/v1
+kind: ApicurioRegistry
+metadata:
+  name: example-apicurioregistry
+spec:
+  configuration:
+    # ...
+    env:
+      - name: REGISTRY_RULES_GLOBAL_VALIDITY
+        value: FULL # One of: NONE, SYNTAX_ONLY, FULL
+      - name: REGISTRY_RULES_GLOBAL_COMPATIBILITY
+        value: FULL # One of: NONE, BACKWARD, BACKWARD_TRANSITIVE, FORWARD, FORWARD_TRANSITIVE, FULL, FULL_TRANSITIVE
+----
 
```

Comment:
Should we also add the new integrity rule to this example? @EricWittmann, I'm assuming that the following config is correct? 
 - name: REGISTRY_RULES_GLOBAL_INTEGRITY
        value: FULL # One of: NONE, NO_DUPLICATES, REFS_EXIST, ALL_REFS_MAPPED, FULL

Reply:
There is a new rule but it's not required in this example. We can consider adding it later.